src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -240,285 +246,316 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
+              <AdminInput
+                name="room.url"
+                autoComplete="off"
+                label={`${ROOM_TAXON.capital} url`}
+                placeholder={`${ROOM_TAXON.capital} url`}
+                register={register}
+                errors={errors}
+              />
+            ) : (
+              // NOTE: Save button doesn't work if the value is missing
+              <input name="room.url" type="hidden" ref={register} />
+            )}
 
-              {room.template &&
-                IFRAME_TEMPLATES.includes(room.template as VenueTemplate) && (
-                  <>
-                    <Form.Label>Livestream URL</Form.Label>
-                    <InputField
-                      name="venue.iframeUrl"
-                      type="text"
-                      autoComplete="off"
-                      placeholder="Livestream URL"
-                      error={errors?.venue?.iframeUrl}
-                      ref={register()}
-                    />
-                    {errors?.venue?.iframeUrl && (
-                      <span className="input-error">
-                        {errors?.venue?.iframeUrl}
-                      </span>
-                    )}
+            <AdminSection
+              title="Select the parent space for the “back” button"
+              withLabel
+            >
+              <SpacesDropdown
+                venueSpaces={venueSpaces ?? ALWAYS_EMPTY_ARRAY}
+                venueId={venueId}
+                setValue={setValue}
+                register={register}
+                fieldName="venue.parentId"
+                defaultSpace={venueValues.parentId}
+              />
+            </AdminSection>
+          </AdminSpacesListItem>
+          <AdminSpacesListItem title="Appearance" isOpened>
+            <AdminSection
+              withLabel
+              title={`${ROOM_TAXON.capital} image`}
+              subtitle="(overrides global settings)"
+            >
+              <ImageInput
+                onChange={changeRoomImageUrl}
+                name="room.image"
+                setValue={setValue}
+                register={register}
+                small
+                nameWithUnderscore
+                imgUrl={room.image_url}
+              />
+              {errors?.room?.image_url && (
+                <span className="input-error">
+                  {errors?.room?.image_url.message}
+                </span>
+              )}
+            </AdminSection>
+
+            <AdminSection
+              withLabel
+              title="Change label appearance"
+              subtitle="(overrides global settings)"
+            >
+              <PortalVisibility
+                getValues={getValues}
+                name="room.visibility"
+                register={register}
+                setValue={setValue}
+              />
+            </AdminSection>
+            {!isLoadingRoomVenue && !!roomVenue && (
+              <>
+                {room.template &&
```

Comment:
Please add this `// @debt use a single structure of type Record<VenueTemplate,TemplateInfo> to compile all these .includes() arrays' flags` in front of all the uses of `.includes()` here in a manner like bellow so that we can repay it in some next PR

```suggestion
                {
                  // @debt use a single structure of type Record<VenueTemplate,TemplateInfo> to compile all these .includes() arrays' flags
                  room.template &&```

Reply:
done 40ab2db